sdk/search/Azure.Search.Documents/src/Batching/SearchIndexingPublisher.cs

```diff
@@ -235,7 +235,7 @@ protected override async Task<bool> OnSubmitBatchAsync(IList<PublisherAction<Ind
         /// <param name="actions">The batch of actions.</param>
         /// <param name="results">The results.</param>
         /// <returns>Actions paired with their result.</returns>
-        private static IEnumerable<(PublisherAction<IndexDocumentsAction<T>>, IndexingResult)> AssociateResults(
+        private static IEnumerable<(PublisherAction<IndexDocumentsAction<T>> Action, IndexingResult Result)> AssociateResults(
```

Comment:
I'm slightly concerned I don't see other changes that reference these values. @tg-msft can you take a look?

Reply:
I suspect deconstruction was used at the consumption points.

Reply:
Yes, we just deconstruct at https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/src/Batching/SearchIndexingPublisher.cs#L169